sdks/godot/addons/com.google.agones/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
+name="Agones SDK"
+description="This is the Godot version of the Agones Game Server Client SDK."
+author="@googleforgames"
+version="1.13.0"
```

Comment:
Does this version need to be kept up to date with agones release versions?

Reply:
The version number is required, but it doesn't necessarily have to correspond with Agones releases.